docs/source/usage.rst

```diff
@@ -482,56 +482,50 @@ configuration, but they must all be passed via the config file.
      - Float
      - Any reasonable real value.
 
-   * - ``solvepp_enabled``
-     - Flag to enable solving for initial partial pressures subject to interior parameters, equilibrium reactions, and melt-vapour partitioning.   
+   * - ``solvevol_use_params``
+     - Flag to enable solving for initial partial pressures subject to interior parameters, rather than using provided initial pressures. 
      - False
      - Integer
      - 0: Disabled, 1: Enabled.
 
-   * - ``T_surf_guess``
-     - Initial guess for surface temperature when ``solvepp_enabled == 1``, units of kelvin.
+   * - ``T_outgas``
+     - Initial guess for surface temperature, units of kelvin.
      - False
      - Float
      - Greater than zero.
    
-   * - ``melt_fraction_guess``
-     - Initial guess for mantle melt fraction when ``solvepp_enabled == 1``.    
+   * - ``Phi_global``
+     - Initial guess for mantle melt fraction.    
      - False
      - Float
      - Between 0 and 1, inclusive.
 
    * - ``CH_ratio``
-     - Initial guess for C/H ratio when ``solvepp_enabled == 1``.    
+     - Required total-planet C/H mass ratio. Used when ``solvevol_use_params == 1``.    
      - False
      - Float
      - Greater than zero.
 
    * - ``hydrogen_earth_oceans``
-     - Total hydrogen inventory of the planet, used when when ``solvepp_enabled == 1``.   Units of Earth oceans equivalent.  
+     - Total hydrogen inventory of the planet. Used when when ``solvevol_use_params == 1``. Units of Earth oceans equivalent.  
      - False
      - Float
      - Greater than zero.
 
    * - ``nitrogen_ppmw``
-     - Initial nitrogen concentration in the mantle when ``solvepp_enabled == 1``. Parts per million of total mantle mass.  
+     - Nitrogen concentration. Used when ``solvevol_use_params == 1``. Parts per million of total mantle mass.  
      - False
      - Float
      - Greater than zero. 
 
    * - ``X_included``
-     - Flag to include or exclude a volatile X from SPIDER and SOCRATES. This value may be overwritten at runtime but will be saved to the helpfile.  
+     - Flag to include X in the model. For H2O, CO2, and N2 this will always equal 1.
      - False
      - Integer
      - 0: Excluded, 1: Included.
 
-   * - ``X_add_bars``
```

Comment:
This is outcommented, meaning this is not possible henceforth?

Reply:
Yes, adding extra bars is not possible alongside the parameterised calculation of partial pressures from C/H, etc. This is because adding these extra bars overspecifies the initial condition, such that these parameters (C/H, etc) lose all meaning. This lead to my confusion with the pure-steam case that I thought was 6 oceans but was actually 30.